config/license-decisions.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - - :name_project
-  - client
+  - ice-ruby
```

Comment:
Or `phraseapp-in-context-editor-ruby`? :)

Reply:
currently using `ice-ruby` as identification (also in google sheets operation is using).